scripts/auto-merge

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/python3
+# SPDX-License-Identifier: Apache-2.0
+
+from github import Github
+import os
+
+# Initialize Github
+github = Github(os.environ['GITHUB_TOKEN'])
+
+for pr in github.search_issues("org:enarx state:open is:public is:pr"):
+    # If all merge criteria pass, merge the PR.
+    if pr.mergeable == True and pr.mergeable_state == "clean":
+        print(f"Merging PR {pr.number} from {repo.name}!")
+        pr.merge(merge_method="rebase")
+    
+    # If not, show what was missing.
+    else:
+        print(f"PR {pr.number} from {repo.name} did not meet the criteria for merge:")
+        print(f"Mergeable? {pr.mergeable}")
+        print(f"Mergeable state: {pr.mergeable_state}")
```

Comment:
This output should be in a table format, one line per PR with fixed-width columns. Columns should be:
1. Single character (merge? check or hash)
2. PR#
3. Mergeable ("mergeable" or "conflicts")
4. State (state text)